src/cookbooks/looker/intro.md

```diff
@@ -0,0 +1,62 @@
+# Introduction to Looker
+
+In 2020, Mozilla chose [Looker](https://looker.com/) as its primary tool for analyzing data.
+It allows data exploration and visualization by experts and non-experts alike.
+
+Access to Looker is currently limited to Mozilla employees and designated contributors. For more information, see [gaining access](../../concepts/gaining_access.md).
+
+## Table of Contents
+
+<!-- toc -->
+
+## Accessing Looker
+
+You can access Mozilla's instance of Looker at [`mozilla.cloud.looker.com`](https://mozilla.cloud.looker.com).
+
+## Getting Started
+
+### Front page
+
+By default, on the front page you will see a list of default folders, which contain links to dashboards.
+These are organized by project.
+Of particular note is the "Mozilla Metrics" folder, which includes several Data-produced and vetted dashboards like the Firefox Corporate KPI Dashboard.
```

Comment:
```suggestion
Of particular note is the ["Mozilla Metrics"](https://mozilla.cloud.looker.com/folders/63) folder, which includes several Data-produced and vetted dashboards like the Firefox Corporate KPI Dashboard.
```

Reply:
Linking is a good suggestion, but we should be probably remove the quotes in that case. Will do that in my branch